src/main/flight/filter.c

```diff
@@ -0,0 +1,27 @@
+/*
+ * filter.c
+ *
+ *  Created on: 24 jun. 2015
+ *      Author: borisb
+ */
+
+#include <stdbool.h>
+#include <stdint.h>
+#include <stdlib.h>
+#include <math.h>
+
+#include "common/axis.h"
+
+#include "flight/filter.h"
+
+extern uint16_t cycleTime;
+
+// PT1 Low Pass filter
+float filterApplyPt1(float input, filterStatePt1_t* state, uint8_t f_cut) {
```

Comment:
@borisbstyle, may I suggest making `dT` a parameter that can be passed to this function? This will allow this function to be reused i.e. for baro or gps that are updated at different rate.